projects/instanceimagenav/requirements.txt

```diff
@@ -0,0 +1,4 @@
+scikit-fmm==2022.8.15
+scikit-learn==0.22.2.post1
+scikit-image==0.15.0
+trimesh
```

Comment:
I had to manually install `natsort` when running the code. Should it be also listed here?